doc/Handlers.md

```diff
@@ -0,0 +1,415 @@
+# Handlers
+
+Each Deku-node exposes several HTTP/TCP endpoints to interact with the chain.
+They are available within [deku_node.ml](../src/bin/deku_node.ml)
+Here are listed each of them, with a `curl` example command to play with them.
+
+Since we use RPC, all these endpoints are POST (only verb available over RPC).
+
+## Prerequisites
+
+In order to be able to play the following `curl` examples, you must have
```

Comment:
Maybe just: `In order to play`

Reply:
Or maybe better with: `In order to run the following ...`

Reply:
Done.